### ISSUE TITLE
Executing centos2ol.sh in CentOS6 - Error: Package: initscripts-9.03.61-1.0.6.el6.x86_64 (@ol6_latest)

### ISSUE THREAD

**Issue by lrico01** [oracle/centos2ol#134]
Hi.
The output of centos2ol.sh is:
...
--> Processing Dependency: /etc/oracle-release for package: initscripts-9.03.61-1.0.6.el6.x86_64
--> Finished Dependency Resolution
Error: Package: initscripts-9.03.61-1.0.6.el6.x86_64 (@ol6_latest)
           Requires: /etc/oracle-release
           Removing: 6:oraclelinux-release-6Server-10.0.2.x86_64 (installed)
               Not found
           Obsoleted By: centos-release-6-10.el6.centos.12.3.x86_64 (C6.10-base)
               Not found
           Available: 6:oraclelinux-release-6Server-9.0.3.x86_64 (ol6_latest)
               Not found
 You could try using --skip-broken to work around the problem
 You could try running: rpm -Va --nofiles --nodigest
Could not automatically sync with Oracle Linux repositories.
Check the output of 'yum distro-sync' to manually resolve the issue.

----
Thanks.

**Comment by lrico01** [comment]
Hi.
I see that "ym -y distro-sync" failed because the script had left a CentOS repo.
I deleted the file in /etc/yum.repo.d and ran "yum -y distro-sync".
Now, from what I've read I'll have to edit the script and run the remaining portion of the code, making sure the variable values are correct.
Thanks.

**Comment by pluto00987** [comment]
I ran into this exact same issue. To add a bit more info, I had a 'CentOS-EoL.repo' file that I had previously created after the 'latest' centos yum repos were all wiped. My file just had entries for the 6.10 repo on vault.centos.org which served to at least allow yum/etc to work.

To work around, I moved the contents of my .repo file into the existing CentOS-Vault.repo file and removed my file. That allowed a second attempt with centos2ol.sh to complete without issue (after having reverted the vm).
